metapackages/auto-instrumentations-node/package.json

```diff
@@ -30,10 +34,10 @@
     "url": "https://github.com/open-telemetry/opentelemetry-js-contrib/issues"
   },
   "peerDependencies": {
-    "@opentelemetry/api": "^1.3.0"
+    "@opentelemetry/api": "^1.4.1"
```

Comment:
I looks like the user needs to install the api package himself. I should expect this package to come bundled with an api dependency. If not, please make it obvious in the README to make people aware

Reply:
moved this to normal dependency to have a more seamless experience for the user.

Reply:
This check can be modified by editing [this script](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/scripts/peer-api-check.js) if needed.

Regardless of the CI, I think it is conceptually correct that this package will bring along with it the@opentelemetry/api dependency, as otherwise, end users will have to install it along with this one which is more room for problems and misunderstandings.